kernel/src/scan/mod.rs

```diff
@@ -627,10 +627,6 @@ mod tests {
 
     #[test]
     fn test_scan_data() {
-        use tracing_subscriber::EnvFilter;
```

Comment:
this was left over that shouldn't be here, and was causing test failures because you shouldn't init in a test like this